client/views/room/contextualBar/RoomFiles/hooks/useFilesList.ts

```diff
@@ -34,6 +34,7 @@ export const useFilesList = (
 	const roomTypes = {
 		c: 'channels.files',
 		l: 'channels.files',
+		v: 'channels.files',
```

Comment:
This endpoint doesn't support voip rooms.

Reply:
How about adding 
```suggestion
		v: 'channels.files',
		
const fetchMessages = useCallback(
		async (start, end) => {
			if (room?.t === 'v') {
				// channels.files is not supported for Voip endpoints.
				return {
					items: [],
					itemCount: 0,
				};
			}
			const { files, total } = await getFiles({
				roomId: options.rid,
				offset: start,
				count: end,
				sort: JSON.stringify({ uploadedAt: -1 }),
				query: JSON.stringify({
					name: { $regex: options.text || '', $options: 'i' },
					...(options.type !== 'all' && {
						typeGroup: options.type,
					}),
				}),
			});

			return {
				items: files,
				itemCount: total,
			};
		},
		[room?.t, getFiles, options.rid, options.text, options.type],
	);		
```

Reply:
Hello Pierre, 
I wont be able to not implement it for 'v' because the compiler throws an error if I do not specify the room type for voip. 
I took a look at the endpoint code app/api/server/v1/channels.js here. I saw that if someone tries to use this endpoint on voip room, server would throw an error.  Which would return an error to the client. I think it is still ok. I think Voip room may have files tomorrow. and amongst the list I feel that channels.files is a better API to be used for voip rooms. I feel that im.files or group.files do not make sense.

The other option is to create another endpoint voip.files and return empty values for time being. @KevLehman Please suggest your thoughts on this.

Reply:
Retyping `roomTypes` would be enough:

```ts
	const roomTypes: Record<IRoom['t'], Extract<PathFor<'GET'>, `${string}.files`>> = {
		c: 'channels.files',
		l: 'channels.files',
		d: 'im.files',
		p: 'groups.files',
	};
```

Reply:
I believe there should be no support for uploaded files for `v`-type rooms, then ignoring it here is enough.

Reply:
I am not sure if I understand this right. Even if I create a room type as @tassoevan is suggesting I am getting following error 
"Property 'v' is missing in type '{ c: "channels.files"; l: "channels.files"; d: "im.files"; p: "groups.files"; }' but required in type 'Record<RoomType, "channels.files" | "groups.files" | "im.files">'."

Reply:
Anyway, I am adding a test case for the voip room in tests/end-to-end/api/02-channels.js